### DISCUSSION TITLE
Projects that could use sgkit

### DISCUSSION THREAD

**Discussion by hammer** [sgkit-dev/sgkit#674] [Ideas]
I'd like to track some repos that might benefit or benefit from `sgkit`. If they're doing population or statistical genetics in Python, drop it here!

**Comment by hammer** [comment]
- https://github.com/hringbauer/hapROH
- https://github.com/jeremymcrae/liftover
- https://github.com/jeremymcrae/ld_estimator
- https://github.com/nadavbra/pwas
- https://github.com/popgenmethods/pyrho

**Comment by alimanfoo** [comment]
I'll add this one - https://github.com/malariagen/vector-data - which has the notebooks backing the MalariaGEN vector data user guide - https://malariagen.github.io/vector-data/landing-page.html. Ultimately it would be great to migrate all of the notebooks to use sgkit instead of scikit-allel.

**Comment by jeromekelleher** [comment]
This is the one I'm most interested in: https://github.com/tskit-dev/tsinfer

Currently tsinfer uses a custom wrapper around zarr, and users have to manually import their data [manually](https://tsinfer.readthedocs.io/en/latest/tutorial.html#data-example). The plan is to remove this layer entirely, and to instead depend on sgkit datasets with the required variables.
